tensorhive/hosts_config.ini

```diff
@@ -1,12 +1,24 @@
-# Example:
-; [example_hostname]
-; user = example_username
+# Example SSH configuration
+# 1. Hosts must be accessible without password (key-based authentication)
```

Comment:
```suggestion
# 1. Hosts must be accessible without password (for example key-based authentication)
```

Reply:
Is there any other way? We **force** using keys anyway, so not using them is:
- a bad idea in general
- problematic for anyone who want set up TensorHive